src/server.js

```diff
@@ -65,6 +65,14 @@ app.get("/presentations", (_, res) => {
   res.status(200).send(html);
 });
 
+app.get("/presentations/:id", (req, res) => { //the id path
+  const presID = req.params.id;
+  const presentation = presentations.find((presentation) => presentation._id == presID);
+  console.log(presentation);
+  const html = render("presentationDesc", {testing: presentation});
```

Comment:
Let's replace `testing` with `presentation`, and update the template as well.

Reply:
sounds good! thanks for the feedback!